src/bundle/Resources/views/themes/admin/account/bookmarks/list.html.twig

```diff
@@ -80,7 +82,7 @@
                 'attr': { 'class': 'ibexa-toggle-btn-state', 'data-toggle-button-id': '#bookmark_remove_remove' }
             }) }}
             {% include '@ibexadesign/ui/component/table/table.html.twig' with {
-                headline: 'bookmark.table.header'|trans|desc('Bookmarks') ~ ' (' ~ pager.count ~ ')',
+                headline: headline(pager.getNbResults()),
```

Comment:
I also think that we should provide the possibility of easy overriding specific headlines to other developers. 
Probably by creating twig blocks which can be overridden in the child view. 
example: https://twigfiddle.com/bonj6s/2